packages/expand-button-react/src/ExpandSection.test.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState, VFC } from "react";
+import { axe } from "jest-axe";
+import userEvent from "@testing-library/user-event";
+import { act, render, screen } from "@testing-library/react";
+import { ExpandSection } from "./ExpandSection";
+
+const onClickSpy = jest.fn();
+
+const ExpandSectionExample: VFC = () => {
+    const [isExpanded, setIsExpanded] = useState(false);
+
+    return (
+        <ExpandSection
+            title={isExpanded ? "Skjul seksjonen igjen" : "Vis den skjulte seksjonen"}
+            onClick={(e, expanded) => {
+                setIsExpanded(expanded);
+                onClickSpy(e);
+            }}
+        >
+            <h2>Skjult seksjon</h2>
+            <p>Her kommer det masse informasjon som er relevant, men ikke så relevant at vi alltid vil vise det.</p>
+            <p>
+                Vi kunne kanskje brukt en Accordion, men vi har ikke <i>flere</i> ting vi vil strukturere på denne
+                måten. Det ser litt rart ut med ett AccordionItem, så da bruker vi heller en ExpandButton
+            </p>
```

Comment:
```suggestion
            <p>
                Vi kunne kanskje brukt en Accordion, men vi har ikke <em>flere</em> ting vi vil strukturere på denne
                måten. Det ser litt rart ut med ett AccordionItem, så da bruker vi heller en ExpandButton
            </p>
```

Ser nå at dette er en test, men kan like så godt være semantisk riktig overalt 😅